packages/create-corvid-app/README.md

```diff
@@ -0,0 +1,14 @@
+# Create Corvid App
+
+`create-corvid-app` allows you to do the necessary setup for using `corvid-cli` and download your [Wix](https://www.wix.com) site in just one command.
+
+## Usage
+
+Assuming you have [Node.JS](https://nodejs.org) version >= 10 installed and working, just open your terminal and run:
+
+```
+npm init corvid-app <any-folder-name> [your-wix-site-url]
+```
+
+Sit back and relax while we initialize the given folder with the recommended setup for working with `corvid-cli` comfortably.
+Assuming you've also provided your wix site url (optional), we will also try to download your wix site into that folder.
```

Comment:
Change to: "If you provide a Wix site URL (optional), we will clone your site into that folder.